spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/mapping/DatastorePersistentEntityImpl.java

```diff
@@ -72,14 +72,6 @@ protected boolean hasTableName() {
 		return this.kind != null && StringUtils.hasText(this.kind.name());
 	}
 
-	@Override
-	public void addPersistentProperty(DatastorePersistentProperty property) {
-		if (!property.isMapped()) {
```

Comment:
It turns out that since `@Transient` already exists, this part is already done by default.